squirrel.frontier/src/test/java/org/dice_research/squirrel/predictor/impl/PredictorImplTest.java

```diff
@@ -0,0 +1,181 @@
+package org.dice_research.squirrel.predictor.impl;
+
+import de.jungblut.math.activation.SigmoidActivationFunction;
+import de.jungblut.math.dense.DenseDoubleVector;
+import de.jungblut.online.regression.RegressionClassifier;
+import de.jungblut.online.regression.RegressionModel;
+import org.dice_research.squirrel.Constants;
+import org.dice_research.squirrel.data.uri.CrawleableUri;
+import org.dice_research.squirrel.predictor.PredictorImpl;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import static org.junit.Assert.assertEquals;
+
+public class PredictorImplTest {
+    private CrawleableUri curi;
+    private PredictorImpl predictor;
+
+    @Test
+    public void train() throws Exception {
+        // //Initialization
+        curi = new CrawleableUri(new URI("https://mcloud.de/web/guest/suche/-/results/search/55?_mcloudsearchportlet_sort=latest"));
+        CrawleableUri curiPos = new CrawleableUri(new URI("https://mcloud.de/export/datasets/037388ba-52a7-4d7e-8fbd-101a4202be7f"));
+        CrawleableUri curiNeg = new CrawleableUri(new URI("https://ckan.govdata.de"));
+
+        predictor = new PredictorImpl();
+        predictor.TRAINING_SET_PATH = "trainDataset.txt";
+
+        // train the learner on two URIs: one RDF and one non RDF
+        predictor.train();
+
+        // predict for a random URI(HTML) example
+        predictor.featureHashing(curi);
+        double pred = predictor.predict(curi);
+        double pround = Math.round(pred*100.0)/100.0;
+        Assert.assertEquals(0.41,pround,0.1);
```

Comment:
What kind of check is this? I do not really believe that you have precalculated which result you expect... therefore, this check is not really helpful.